### PR TITLE
Feature/merge r35 to r36

### PR DESCRIPTION
**Pull request by CuriousVini**
`cdap-docs/reference-manual/source/release-notes.rst` - version conflict, merged changes from release/3.6 and release/3.5
`cdap-docs/admin-manual/build.sh` - MD5 Hash, kept changes from release/3.6

Build: http://builds.cask.co/browse/CDAP-RUT246-4

### CODE REVIEW

File: cdap-docs/reference-manual/source/release-notes.rst [hunk 1]
```diff
@@ -23,6 +23,26 @@ Cask Data Application Platform Release Notes
    :backlinks: none
    :depth: 2
 
+`Release 3.6.1 <http://docs.cask.co/cdap/3.6.1/index.html>`__
+=============================================================
+
+- :cask-issue:`CDAP-7250` - Fixed an issue where dataset usage was not being recorded
```
Comment (awholegunch) [inline cdap-docs/reference-manual/source/release-notes.rst]: This issue was fixed in 3.6.0; so it should be removed from here and replaced where it was.

**Comment by awholegunch** [comment]
Doc changes LGTM

**Review by bdmogal — APPROVED** [review]
lgtm

**Comment by CuriousVini** [comment]
Build is green. Merging.

**Pull request merged**